index.html

```diff
@@ -815,95 +912,128 @@ <h3>
-      </p>
+        <p>
+          It is a common mistake to claim that "JSON-LD is just JSON", and to
+          justify injecting the `@context`
+          representation-foreign entry into a DID document in the
+          `application/did+json` representation. This claim
+          however is true only on the syntax level, but wrong and dangerous on
+          the semantic level, especially when
+          security-related properties are involved. In the example above,
+          different consumer implementations are likely
+          to
+          come to different conclusions on how the DID can be controlled.
+        </p>
 
-      <p class="advisement">
-        Representation-foreign entries in DID documents should be avoided, since they can lead to inconsistent and
-        unpredictable DID control decisions.
-      </p>
+        <p class="advisement">
+          Representation-foreign entries in DID documents should be avoided,
+          since they can lead to inconsistent and
+          unpredictable DID control decisions.
+        </p>
 
-    </section>
+      </section>
 
-    <section class="informative">
-      <h3>
-        Conversion between Representations
-      </h3>
+      <section class="informative">
+        <h3>
+          Conversion between Representations
+        </h3>
 
-      <p>
-        The DID document data model and the production/consumption rules of representations have been designed to enable
-        lossless conversion between representations. Conversion between representations is achieved by executing the
-        consumption rules of the source representation, and then the production rules of the target representation.
-      </p>
+        <p>
+          The DID document data model and the production/consumption rules of
+          representations have been designed to
+          enable
+          lossless conversion between representations. Conversion between
+          representations is achieved by executing the
+          consumption rules of the source representation, and then the
+          production rules of the target representation.
+        </p>
 
-      <p>
-        The <a href="https://w3c.github.io/DID-SPEC-REGISTRIES/">DID Specification Registries</a> provide additional
-        information about properties as well as representation-specific entries that help with such conversion. For
-        example, a DID document in the `application/did+json` representation can be converted to the
-        `application/did+ld+json` representation by adding a `@context` representation-specific entry during production,
-        using the JSON-LD context information in the DID <a href="https://w3c.github.io/DID-SPEC-REGISTRIES/">Specification
-        Registries</a>.
-      </p>
+        <p>
+          The <a href="https://w3c.github.io/did-spec-registries/">DID
+            Specification Registries</a> provide additional
+          information about properties as well as representation-specific
```

Comment:
This should be changed to `https://www.w3.org/TR/did-spec-registries` (through a proper external reference). It is not appropriate to refer to the editor's draft if there is an official /TR document.